test/integration/test_idr/test_lib_api.py

```diff
@@ -0,0 +1,40 @@
+from omero.gateway import BlitzGateway
+from rdflib import Literal, URIRef
+
+from omero_rdf import RdfLibrary
+
+
+def test_rdf_library_returns_expected_triples():
+    with BlitzGateway(
+        username="public",
+        passwd="public",
+        host="idr.openmicroscopy.org",
+        port="4064",
+        secure=True,
+    ) as conn:
+        lib = RdfLibrary(conn)
```

Comment:
I find "RdfLibrary" misleading as a name. How about "Triplyfier" or sth. similar, more expressive about what it's actually doing? It is a library, all right but that information does not have to be in the name IMHO. Just a minor thought.